application/classes/Ushahidi/Core.php

```diff
@@ -511,6 +511,19 @@ public static function init()
 			'post_value_factory' => $di->lazyGet('repository.post_value_factory'),
 			'post_value_validator_factory' => $di->lazyGet('validator.post.value_factory'),
 			];
+
+			$di->params['Ushahidi_Validator_Post_Import'] = [
```

Comment:
I think this is unnecessary. AuraDI can figure out the params needed from the parent class